File: IDE/temp files/newfile638232.dart


// Print function for various types

int HTVM_Size<T>(List<T> arr) {
  return arr.length;
}


void main(List<String> arguments) {
    List<int> arr123 = [1, 3, 5];
    for (int A_Index1 = 0; A_Index1 < HTVM_Size(arr123) + 0; A_Index1++) {
        print(arr123[A_Index1]);
    }
    

}

File: IDE/temp files/newfile638232.rb


# Print function for various types
def print(value)
  if value.is_a?(String)
    puts value
  elsif value.is_a?(Numeric)
    puts value.to_s
  elsif value.is_a?(TrueClass) || value.is_a?(FalseClass)
    puts value ? "true" : "false"
  else
    puts "Unsupported type"
  end
end

def HTVM_Size(arr)
  arr.size
end


arr123 = [1, 3, 5]
for a_index1 in 0...HTVM_Size(arr123) + 0 do
    print(arr123[a_index1])
end

